### ISSUE TITLE
Webview does not have momentum scrolling on iPhone

### ISSUE THREAD

**Issue by aflanagan** [facebook/react-native#5447]
When scrolling a webview the scrolling stops immediately after removing your finger from the screen, as opposed to the normal "momentum" style scrolling. It works fine on an iPad, but does not work on an iPhone.

React Native 0.17.0
iOS Versions Tested: 8.4, 9.1, 9.2
Devices Tested: iPhone 6, iPhone 6s, iPad Retina, iPad Mini

```js
import React, { WebView } from 'react-native'

class App extends React.Component {

  render() {
    return (
        <WebView
          url={'https://news.ycombinator.com'}
          />
    )
  }
}
React.AppRegistry.registerComponent('App', () => App);
```

**Comment by aflanagan** [comment]
After looking at the implementation of RCTWebView I added
```
 _webView.scrollView.decelerationRate = UIScrollViewDecelerationRateNormal;
```

to 

```
- (instancetype)initWithFrame:(CGRect)frame
{
  if ((self = [super initWithFrame:frame])) {
    super.backgroundColor = [UIColor clearColor];
    _automaticallyAdjustContentInsets = YES;
    _contentInset = UIEdgeInsetsZero;
    _webView = [[UIWebView alloc] initWithFrame:self.bounds];
    _webView.delegate = self;
    [self addSubview:_webView];
  }
  _webView.scrollView.decelerationRate = UIScrollViewDecelerationRateNormal;
  return self;
}
```

that fixes the problem, but i'm unsure of the ramifications of this change. Happy to submit a PR if y'all think this is the correct solution.

**Comment by ide** [comment]
This is how WebViews on iOS operate. We can expose decelerationRate but shouldn't set it to Normal since there are glitches.

**Comment by aflanagan** [comment]
@ide interesting. I'd be happy to do the work to expose that. What would you recommend setting it to, and what are the possible glitches that would result from having it set to Normal?

**Comment by ide** [comment]
Mozilla told me they saw issues with setting it to Normal. Leaving it at the default is best for the general case.

**Comment by aflanagan** [comment]
@ide the reason I really want to set it to normal is that i'm building a section of an app that has a lot of articles. These are already published on the web and we want to be able to reuse those in the app. Having smooth scrolling on them feels a lot better in the app!

I've only been testing this change since last night, but I haven't noticed any problems on iPhone 5s, 6, 6s.

If you're open to a PR that exposes a flag for changing this setting I'm happy to submit one.

**Comment by ide** [comment]
RN should let you customize the setting -- the point I meant to make was that we should preserve the system default for stability reasons unless you opt-in to override it. (And I agree the default feels crappy though...)

> If you're open to a PR that exposes a flag for changing this setting I'm happy to submit one.

Yes please do.

**Comment by aflanagan** [comment]
Thanks! I'll submit one shortly.

**Comment by aflanagan** [comment]
closed by #5527
